### PR TITLE
Add flash.display.PixelSnapping enum class

### PR DESCRIPTION
**Pull request by rameshvarun**
[`flash.display.PixelSnapping`](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/PixelSnapping.html) is a simple enum class that contains three string constants "always", "auto", and "never".

I implemented this following the same pattern as [`flash.display.LineScaleMode`](https://github.com/ruffle-rs/ruffle/blob/79deba87f3d8e93cc990f657b14ca3af396afc5a/core/src/avm2/globals/flash/display/linescalemode.rs).

### CODE REVIEW

**Comment by relrelb** [comment]
Can you please run `cargo fmt --all` to apply Rust formatting? Otherwise looks good to me.

**Comment by relrelb** [comment]
Due to CI issues, can you please rebase your changes onto the latest master?

File: core/src/avm2/globals/flash/display/pixelsnapping.rs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        QName::new(Namespace::package("flash.display"), "PixelSnapping"),
+        Some(QName::new(Namespace::public(), "Object").into()),
+        Method::from_builtin(instance_init, "<PixelSnapping instance initializer>", mc),
+        Method::from_builtin(class_init, "<PixelSnapping class initializer>", mc),
+        mc,
+    );
+
+    let mut write = class.write(mc);
+
+    write.set_attributes(ClassAttributes::SEALED);
```
Comment (adrian17) [inline core/src/avm2/globals/flash/display/pixelsnapping.rs:46]: Looking at docs, I think this should be `final` too.

**Review by adrian17 — APPROVED** [review]
Looks good :)

**Review by relrelb — APPROVED** [review]

**Pull request merged**